.devcontainer/devcontainer.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "pg-latex",
+  "dockerComposeFile": ["docker-compose.yaml"],
+  "service": "latex",
+  "runServices": ["latex"],
+  "shutdownAction": "stopCompose",
+  "workspaceFolder": "/workspace",
+  "postCreateCommand": "pre-commit install || echo 'Warning: pre-commit installation failed'",
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "james-yu.latex-workshop",
+        "eamodio.gitlens",
+        "shardulm94.trailing-spaces",
+        "stkb.rewrap",
+        "vscode-icons-team.vscode-icons",
+        "MS-vsliveshare.vsliveshare",
+        "streetsidesoftware.code-spell-checker",
+        "streetsidesoftware.code-spell-checker-polish",
+        "usernamehw.errorlens"
+      ],
+      "settings": {
+        "files.eol": "\n",
+        "latex-workshop.linting.chktex.enabled": true,
+        "latex-workshop.linting.chktex.exec.path": "chktex",
+        "latex-workshop.latex.clean.subfolder.enabled": true,
+        "latex-workshop.latex.autoClean.run": "onBuilt",
+        "editor.formatOnSave": true,
+        "files.associations": {
+          "*.tex": "latex"
+        },
+        "latex-workshop.formatting.latexindent.path": "latexindent",
```

ciągle mam ten błąd z `Please set your LaTeX formatter in 'latex-workshop.formatting.latex*.`

Ta zmiana to naprawia (patha nie trzeba ustawiać, więc wystarczy zamienić):

```suggestion
        "latex-workshop.formatting.latex": "latexindent",
```

EDIT: ogarnąłem już tę zmianę, dziwne jakby to coś psuło, więc zaraz merguję PRa z tym jak jest teraz.

---

Ciekawe, bo u mnie ten błąd się nie pojawiał. Poza tym wszystko działa :)